libs/sdk-ui-dashboard/src/_staging/dashboard/flexibleLayout/layoutConfiguration.ts

```diff
@@ -23,3 +26,49 @@ export function getLayoutConfiguration(layout: IDashboardLayout<ExtendedDashboar
         },
     };
 }
+
+/**
+ * Determines the direction of a container in a dashboard layout based on the provided layout configuration
+ * and item path. The direction specifies whether child items within the container are laid out in a row
+ * or column.
+ *
+ * @param layout - The dashboard layout containing the structure and configuration of widgets and containers.
+ * @param itemPath - The path to a specific layout item within the dashboard layout structure.
+ *  If undefined, the function assumes a default "row" direction.
+ *
+ * @returns The direction of the container ("row" or "column").
+ */
+export const getContainerDirectionAtPath = (
```

Comment:
These two functions seems very similar and are just a syntactic sugar over the original function in this file. I needed them in various places and moved them here. It's not easy to unify them into one function.

It would be nice to be able to rewrote whole flexible layout from top to bottom to use facades and never touch LayoutItems or ExtendedWidget types. This way lots of stuff is done multiple ways because you don't have always the same level of information.